api/v1alpha1/httproute_types.go

```diff
@@ -264,6 +264,45 @@ type ForwardToTarget struct {
 	//
 	// +optional
 	TargetPort *TargetPort `json:"targetPort" protobuf:"bytes,2,opt,name=targetPort"`
+
+	// Timeout specifies the maximum time to wait for a response after sending
+	// an HTTP request to targetRef.
+	//
+	// Support: Core
+	//
+	// +optional
+	Timeout *metav1.Duration `json:"timeout,omitempty" protobuf:"bytes,3,opt,name=timeout"`
```

Comment:
It seems this is read timeout, should we rename this to ReadTimeout?
We should be explicit here if this is TTFB (time to first byte) or not. It seems most proxies have this meaning.

Do we also need Connect Timeout for time to define timeout for the TCP handshake?

Reply:
PTAL at my latest changes and let me know if you feel this is a good starting point for connection timeouts and retries.